fairworkflows/fairstep.py

```diff
@@ -3,13 +3,16 @@
 import warnings
 from typing import List
 from urllib.parse import urldefrag
-import warnings
+
 import rdflib
 from rdflib import RDF, DCTERMS
 
 from .nanopub import Nanopub
 from .rdf_wrapper import RdfWrapper
 
+FAIRSTEP_PREDICATES = [RDF.type, Nanopub.PPLAN.hasInputVar,
```

Comment:
I suppose at some point we will want to move this sort of thing into an effective "config" file? To allow the library to be configured to use a different ontology for describing the workflow. For example, ```cwltool``` will dump RDF triples describing its workflows if asked to, so perhaps it would be one way to test the generality of the tool if you could configure it to read in those triples as a ```FairWorkflow```. Obviously this is beyond the scope of this PR, but I was wondering if perhaps we should open an issue for it?

Reply:
With config "file" you mean a file that configures the behavior of the library? I am very much pro making the RDF-format configurable (instead of only supporting plex), but as a user I would rather like to control that directly when interacting with the library (i.e. pass format='plex' to from_rdf method of FairWorkflow). My gutfeeling is that there will be subtle differences between different workflow RDF formats with the consequence that we can not use the FairWorkflow class as a unified representation of all workflow RDF formats. My feeling would be that we would need different classes (i.e. PlexWorkflow, CwlWorkflow, that subclass from a universal FairWorkflow class). But I agree we can open an issue for this, and continue the discussion there!

Reply:
@raar1 do you open this issue?